### ISSUE TITLE
Relative positioning of monitor

### ISSUE THREAD

**Issue by Fymyte** [hyprwm/Hyprland#2377]
Add relative position of monitors in config.

It could be easier to specify a position relatively to another monitor, instead of an absolute position value.
A feature similar to xrandr `--left-of`, `--right-of`, `--above`, `--bellow` 

The syntax would be
```hypr
monitor = [monitor], <resolution>, direction:<monitor>, <scale>
```
Example:
```hypr
monitor = DP-1, preferred, auto, auto
monitor = DP-2, preferred, above:DP-1, auto
```
with the options
- `above`
- `bellow`
- `left`
- `right`

**Comment by vifino** [comment]
This is something I long for. I connect my laptop to many different screens with different resolutions, aspect ratios, scale and positioning.
"Correct" positioning is very hard because of that.

**Comment by jficz** [comment]
there's no need to implement this in Hyprland, you can use [kanshi](https://wayland.emersion.fr/kanshi/) to achieve exactly the same thing

**Comment by jficz** [comment]
@Lillecarl do you mean #2586 ? That one is only related to a rather specific use case and is imo caused by a Hyprland bug, not directly related to Kanshi.

I'm using Kanshi with Hyprland and I don't have any issues at all. As long as you don't disable your primary display on a laptop (or do something similar), you should be ok, too.

**Comment by Lillecarl** [comment]
It wasn't that specific one, it was a reply in another issue related to display mgmt. I can't recall what I was searching for when I found it. I figured I'd mention that there might be footguns and it's not supported (YMMV).

I'll give Kanshi a shot myself, got 3 different setups I'm commuting between. Thanks

**Comment by MightyPlaza** [comment]
maybe checkout nwg-displays as well

**Comment by Lillecarl** [comment]
@MightyPlaza I gave nwg-displays a shot while troubleshooting manual positioning yesterday (fixed in https://github.com/hyprwm/Hyprland/commit/7c4daee29a353fa7e800c4874ce6d491eda458c3). It was pretty nice as it also spat out the configuration rows it applied. However after said fix I configured Kanshi and it configured my displays correctly at work when I popped the USB-C in this morning.

@vaxerski Any particular reason Kanshi and nwg-displays is not mentioned under https://wiki.hyprland.org/Configuring/Monitors/ as an alternative? 😄
